saleor/graphql/checkout/types.py

```diff
@@ -187,7 +211,14 @@ class Checkout(CountableDjangoObjectType):
     shipping_method = graphene.Field(
         ShippingMethod,
         description="The shipping method related with checkout.",
+        deprecation_reason="Use delivery method instead",
+    )
+
+    delivery_method = graphene.Field(
+        DeliveryMethod,
+        description="The shipping method related with checkout, or warehouse if C&C",
```

Comment:
If we add a description to the `DeliveryMethod` union, here we could simply say "The delivery method selected for this checkout".

Reply:
Done.